Mathlib/Algebra/Order/Group/Pointwise/Interval.lean

```diff
@@ -450,6 +450,38 @@ theorem image_sub_const_Ioc : (fun x => x - a) '' Ioc b c = Ioc (b - a) (c - a)
 theorem image_sub_const_Ioo : (fun x => x - a) '' Ioo b c = Ioo (b - a) (c - a) := by
   simp [sub_eq_neg_add]
 
+lemma mapsTo_const_sub_Ici : MapsTo (fun x ↦ a - x) (Ici b) (Iic (a - b)) := by
+  rw [←Set.image_const_sub_Ici]
```

Comment:
nit:
```suggestion
  rw [← Set.image_const_sub_Ici]
```
throughout

Reply:
Wait, we had a linter for that. Where did it go?